Library/brew.rb

```diff
@@ -50,6 +50,20 @@
   EOABORT
 end
 
+if OS.mac? && MacOS.version == "10.11" && MacOS::Xcode.version < "7.0"
```

Comment:
Need a more complex conditional here - we need to check for the cases where:

* Xcode is installed, but below 7
* Xcode isn't installed, but CLT 7 is installed
* Xcode and CLT are both installed, both out of date
* Xcode and CLT are both installed, only one is up to date

This should probably also be part of the Xcode preinstall checks in `cmd/install` rather than here: https://github.com/Homebrew/homebrew/blob/master/Library/Homebrew/cmd/install.rb#L112-L123